### PR TITLE
docs: update quickfix section

### PR DESCRIPTION
**Pull request by nix6839**
Support custom code action menu shortcut on VSCode >= 1.71.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,32 +261,33 @@ See gif in action:
...
 ```
 
 and add to your init.vim:
 
 ```vim
-nnoremap z= <Cmd>call VSCodeNotify('keyboard-quickfix.openQuickFix')<CR>
+nnoremap z= <Cmd>call VSCodeNotify('editor.action.quickFix')<CR>
 ```
 
+You can now use <kbd>z=</kbd> to open the code action menu, and <kbd>C-n</kbd> and <kbd>C-p</kbd> to navigate it.
```
Comment (justinmk) [inline README.md:289]: this is a markdown doc, why not use markdown?
```suggestion
You can now use `z=` to open the code action menu, ...
```

Comment (nix6839) [reply README.md:289]: Visual difference and markdown allows inline html. See <https://daringfireball.net/projects/markdown/syntax#html>
image

Comment (nix6839) [reply README.md:289]: And I want to ask. Do `<code>` and `<kbd>` mean the same thing? You can see following:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code

Comment (theol0403) [reply README.md:289]: We should remove that section entirely from readme and add to package.json bindings.

**Comment by theol0403** [comment]
We should remove that section entirely from readme and add to package.json bindings.

**Comment by nix6839** [comment]
I agree! I'll close the issue.

**Pull request closed without merging**